src/main/java/tempest/Module.java

```diff
@@ -158,6 +173,15 @@ public String getName() {
         return name;
     }
 
+    /**
+     * Gets the weekly goal of the module.
+     *
+     * @return The weekly goal for the module
+     */
+    public int getWeeklyGoal(){
+        return weeklyGoal;
+    }
```

Comment:
Because we can't initialise the weeklyGoal variable, if it's not been set, it has a value of null - which throws an error because it can't be returned. So return 0 if weeklyGoal is null